content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -33,13 +33,15 @@ You can access performance metrics for your views in:
 ## Event timings and core web vitals
 
 <div class="alert alert-warning">
-  Datadog's core web vitals metrics are available from the <a href="https://github.com/DataDog/browser-sdk">@datadog/browser-rum</a> package v2.2.0+.
+  Datadog's Core Web Vitals metrics are available from the <a href="https://github.com/DataDog/browser-sdk">@datadog/browser-rum</a> package v2.2.0+.
 </div>
 
 [Google's Core Web Vitals][5] are a set of three metrics designed to monitor a site's user experience. These metrics focus on giving you a view of load performance, interactivity, and visual stability. Each metric comes with guidance on the range of values that translate to good user experience. Datadog recommends monitoring the 75th percentile for these metrics.
 
 {{< img src="real_user_monitoring/browser/core-web-vitals.png" alt="Core Web Vitals summary visualization"  >}}
 
+Real User Monitoring includes these metrics as lab metrics in a RUM application's Overview page.
```

Comment:
Ah so it's Synthetics that had two of the metrics available as lab ('controlled' environment) metrics, the RUM metrics for CWV are all real metrics, sorry for the confusion!